### PR TITLE
Minor fix product names capitalisation for readability

### PR DESCRIPTION
**Pull request by gonzalo-bulnes**
## Status

Ready for review

## Description of Changes

This is a follow up on https://github.com/freedomofpress/securedrop/pull/5447 which fixes the case of the following product names: Tor, Tor Browser, Tails, SSH, Apache.

As far as I can tell, none of these strings are internationalized.
Some of them are public-facing (e.g. docs), others are code or configuration comments, other would likely be visible to _admins_ (comments of iptables rules).

**Considerations**

- Sometimes, specially in configuration comments, names could arguably refer to the executable name **or** the program name (e.g. `# start ssh service`). I opted for the comment to say `# start SSH service` when I saw that some comments would say `apache` (`Apache`) even when the executable name was actually `apache2`.

- In general, I assumed that comments were for human consumption and biased towards using product names when both the product and the executable name were an option.

- I kept commits granular in order to allow for cherry-picking. ~~(In particular, I'm not sure that touching the _iptables_ rules to fix the comment case is an obviously good idea auditability-wise, so I kept those changes in three separate commits.)~~

## Testing

- [ ] Make sure nothing was modified that results in commands being executed differently. ~~(iptables comments are arguably part of the command!)~~
- [ ] Read the comments and documentation text to confirm that they make sense.

## Deployment

- ~~:warning: If there are processes (or documentation) that depend on the exact wording/case of iptables comments, please call it out.~~ 
- [ ] Two strings are internationalized. I didn't modify the .pot file because I assumed strings would be extracted following a dedicated process.

### If you made changes to the server application code:

- [ ] Linting (`make lint`)  pass in the development container (I didn't run it locally)
- [x] Tests (`make test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [x] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass (I didn't run it locally)

### If you made non-trivial code changes:

_See callout in the **deployment** section._

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

**Comment by gonzalo-bulnes** [comment]
**Note**: `make test` locally had one failure (**test_login_and_logout**). I haven't investigated further at this point, but it looks like it may be a legit failure. 

The document body was truncated in the test output, but I think the test update may belong with [this change](https://github.com/freedomofpress/securedrop/pull/5447/files#diff-9038818d9dc776fb7530c4d6d3454d0dR7) in https://github.com/freedomofpress/securedrop/pull/5447. (And maybe my idea of separating the two pull requests wasn't the best approach.)

```
tests/test_source.py::test_login_and_logout FAILED

assert 'This will clear your Tor Browser activity data' in '<!DOCTYPE html>\n<html lang="en" dir="ltr">\n  <head>\n    <meta charset="utf-8">\n    <meta name="viewport" content=...">\n    SecureDrop is a project of Freedom of the Press Foundation.\n  </div>\n</footer>    </div>\n  </body>\n</html>'
```

**Edit**: this is fixed, I moved the test update to the PR where the internationalized strings were updated.

**Comment by gonzalo-bulnes** [comment]
Note: there is [one failing step in the Circle CI build](https://app.circleci.com/pipelines/github/freedomofpress/securedrop/739/workflows/3d4012e2-795d-4ca4-beda-91be66567002/jobs/43350) that I believe to be the one that runs `make test`.

For what is worth, I confirm `make test` passes locally.

I couldn't re-run the workflow (all options appear disabled to me), but I'll re-trigger a build by fixing one more thing since I'm looking at Tor Browser strings.

**Review by rmol — DISMISSED** [review]
Approving based on visual review. Someone whose setup currently supports the configuration tests should run those before merging.

File: install_files/ansible-base/roles/restrict-direct-access/templates/rules_v4
Comment (zenmonkeykstop) [inline install_files/ansible-base/roles/restrict-direct-access/templates/rules_v4]: As this file is updated only when the install playbook is run, its changes will not be reflected on many production systems. This means that there will be configuration drift (albeit comments-only) between existing and new instances that may complicate support and automated tests. IMO it would be worth deferring these changes until the next mandatory install playbook run (currently the trusty->focal migration, but could happen before then).

File: molecule/testinfra/staging/app/iptables-app-staging.j2
Comment (zenmonkeykstop) [inline molecule/testinfra/staging/app/iptables-app-staging.j2]: Same point as above for the prod iptables template.

File: molecule/testinfra/staging/mon/iptables-mon-prod.j2
Comment (zenmonkeykstop) [inline molecule/testinfra/staging/mon/iptables-mon-prod.j2]: Same point as above for the prod iptables template.

File: molecule/testinfra/staging/mon/iptables-mon-staging.j2
Comment (zenmonkeykstop) [inline molecule/testinfra/staging/mon/iptables-mon-staging.j2]: Same point as above for the prod iptables template - if it's not updated then these shouldn't be either.

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]
Thanks for this extensive readability fix! I have some concerns re: updates to iptables rules templates and associated tests as per comments, but otherwise it's looking good.

File: docs/deployment/landing_page.rst
Comment (eloquence) [inline docs/deployment/landing_page.rst]: Nit: if we're capitalizing the product name, we should IMO not also use the definite article (`the`), i.e. I would suggest changing such instances to "may not be using Tor Browser yet", in the same way you would say "may not be using Firefox" instead of "may not be using the Firefox".

**Comment by zenmonkeykstop** [comment]
As per @rmol's [comment](https://github.com/freedomofpress/securedrop/pull/5448#pullrequestreview-466817440), testinfra tests are passing with no changes.

File: install_files/ansible-base/roles/restrict-direct-access/templates/rules_v4
Comment (gonzalo-bulnes) [reply install_files/ansible-base/roles/restrict-direct-access/templates/rules_v4]: This makes a lot of sense to me, I suspected the iptables rules had potential to be a special case _somehow_.

File: docs/deployment/landing_page.rst
Comment (gonzalo-bulnes) [reply docs/deployment/landing_page.rst]: I'll remove the articles in front of "Tor Browser" :+1:

**Review by gonzalo-bulnes — COMMENTED** [review]
- I removed all changes in the `install_files/` directory, as per @zenmonkeykstop's [comment](https://github.com/freedomofpress/securedrop/pull/5448#discussion_r470002207). (:bulb: TIL)

- And I reviewed the use of the article 'the' in front of "Tor Browser" as per @eloquence' [comment](https://github.com/freedomofpress/securedrop/pull/5448#discussion_r470052606). (I hadn't thought of that either!)

:warning: Heads up:

- There are fewer files changed overall, because of removing the `install_files/` changes, but the article review touched some files previously untouched.
- I removed one `the` in internationalized strings in the _source interface_. (But didn't modify the `.pot` file, same as in #5447)

File: install_files/ansible-base/roles/restrict-direct-access/templates/rules_v4
Comment (gonzalo-bulnes) [reply install_files/ansible-base/roles/restrict-direct-access/templates/rules_v4]: I removed all changes to the `install_files/` directory. https://github.com/freedomofpress/securedrop/pull/5448#pullrequestreview-467929827

File: molecule/testinfra/staging/mon/iptables-mon-prod.j2
Comment (gonzalo-bulnes) [reply molecule/testinfra/staging/mon/iptables-mon-prod.j2]: :+1:

File: molecule/testinfra/staging/mon/iptables-mon-staging.j2
Comment (gonzalo-bulnes) [reply molecule/testinfra/staging/mon/iptables-mon-staging.j2]: :+1:

File: molecule/testinfra/staging/app/iptables-app-staging.j2
Comment (gonzalo-bulnes) [reply molecule/testinfra/staging/app/iptables-app-staging.j2]: :+1:

**Review by zenmonkeykstop — APPROVED** [review]
Revised changes look good based on visual review, `make testinfra` passes, and string change is present in SI. This is good to go IMO once CI passes (there was an unrelated `staging-test-with-rebase` flake).

**Comment by eloquence** [comment]
Just a note that we should squash those 10 commits, this can be done by the maintainer who merges the change.

**Comment by gonzalo-bulnes** [comment]
I'm happy to squash the commits as well @eloquence  (not right now, so anyone feel free to do it if you prefer!)

**Comment by zenmonkeykstop** [comment]
D'oh, forgot to squash, gonna leave it as is so as to avoid force-pushes on develop.

**Comment by gonzalo-bulnes** [comment]
Oops. Thanks @zenmonkeykstop and everyone!

**Pull request merged**
